src/advanced/bitcoin-anchoring.md

```diff
@@ -334,10 +294,9 @@ The service provides the following public API endpoints:
 - [Get the actual anchoring address](#actual-address)
 - [Get the following anchoring address](#following-address)
 - [Get an anchoring transaction](#get-an-anchoring-transaction)
-- [Get a proof for an Exonum block](#cryptographic-proof-for-an-exonum-block)
 
 All REST endpoints share the same base path, denoted **{api_prefix}**,
```

Comment:
We've generally moved to describing HTTP API in the corresponding crate, because it's easier to keep it updated and test it this way. Do I understand correctly that testing the anchoring service endpoints that way would require non-trivial effort?

Reply:
Anchoring crate already has API endpoints description, but not such verbose as here.